doc/user-manual/tools/literate-programming.rst

```diff
@@ -161,6 +161,13 @@ Literate Forester
 Files ending in :file:`.lagda.tree` are interpreted as literate
 Forester_ files. Literate forester use ```\agda{...}``` for code blocks.
 
+* You will need the postprocessor agda-tree_ to convert ``html/*.tree`` to a valid forester tree.
+
+  * ``agda --html --html-highlight=code example.lagda.tree`` will produce the file ``html/example.tree``.
+  * Run ``agda-tree build`` at where ``html/`` is located, this will produce subdirectory ``trees/`` there.
+  * Add ``trees/`` to ``forest.toml``.
+  * Add ``html/`` to ``forest.toml`` as assets.
```

Comment:
This does not seem sufficient.
`Agda.css` is not linked from any `.xml` file.

We are doing this back-and-forth here... I complain that a step is missing, you add a step, I still have to complain etc.  Not very efficient.

Can you maybe start over, in a fresh directory, and record every step that is needed to get the final result, so that we arrive at complete instructions?

Reply:
How the process missing? Which forester do you use?

Reply:
I see, so forester use `theme` which located at https://git.sr.ht/~jonsterling/forester-base-theme, you will need to modify its `tree.xsl` (add another line beside `<link rel="stylesheet" href="style.css" />` to get others css supports).